public/js/cloud_share_modal.js

```diff
@@ -117,25 +117,24 @@ export default class CloudShareModal extends React.Component {
 
     return(
       <form onSubmit={this.handleSubmit}>
-        <div className="form-group">
-          <label htmlFor="emailInput">Your Email Address</label>
+        <div className="px-6 mb-3">
+          <label htmlFor="emailInput" className="text-seqblue hover: text-orange cursor-pointer mb-0">Your Email Address</label>
```

Comment:
I think you had a bad regexp/paste here and in many other places `hover: text-orange` -> `hover:text-orange`

Reply:
updated